src/angular.headroom.js

```diff
@@ -1,37 +1,37 @@
-(function(angular) {
-
-  if(!angular) {
-    return;
```

Comment:
Would it be worth leaving this check and doing a `console.warn` before the return statement?